### DISCUSSION TITLE
How do I know if the onBoundsChanged was triggered by the user or programatically?

### DISCUSSION THREAD

**Discussion by juliomuhlbauer** [visgl/react-google-maps#437] [Q&A]
I have a function that I want to only be triggered when the interacts with the map (zoom, dragging).
But the onBoundsChanged is triggered when I set the zoom or bounds programmatically via hooks too.
I tried filtering with domEvent parameter, but couldn't get any value.

**Comment by zy-jtp** [comment]
I might be wrong but I guess you can only use `dragend` event to identify a user action. However bounds change also through zooming so you only have `zoom_change` there. I put a control on the map whenever the zoom changes so I can trigger requests only through that button.
